cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.50.0"
+__version__ = "2.50.1"
```

Comment:
Shouldnt we bump the major or minor rather than the patch? See https://semver.org/spec/v2.0.0.html

Reply:
Nice catch. Looking at previous feature releases (e.g. this [one](https://github.com/cognitedata/cognite-sdk-python/pull/858)https://github.com/cognitedata/cognite-sdk-python/pull/858) it should be minor

Reply:
[9f6262b](https://github.com/cognitedata/cognite-sdk-python/pull/928/commits/9f6262b19a5dfd8c454d9811c10b42ebc8f6eeba)